### PR TITLE
Create related Reserved page files.

### PR DESCRIPTION
**Pull request by yumeR07**
The following files have been modified.

1. HomeController
To show "reserved.show_details"page and "reserved.join_users"page.
2. User.php
Add "use App\Models\Participant".
3. modal.scss
Fix to minimum height.
4. top.blade.php
Add "/" . With this addition, the category image now appears.
5. join_users.blade.php
New file.
6. show_details.blade.php
New file.
7. web.php
Add line72 to show "join_users.blade.php"

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -18,7 +19,7 @@ class HomeController extends Controller
      *
      * @return void
      */
-    public function __construct(Category $category, User $user, Meeting $meeting)
+    public function __construct(Category $category, User $user, Meeting $meeting,)
```
Comment (krs91tk) [inline app/Http/Controllers/HomeController.php:22]: you need this ','?

File: resources/views/users/top.blade.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
                 <div class="card col-lg-3 col-md-6 mb-4 p-0">
                     <a href="" class="">
                       <div class="card-top" style="height: 156.31px; background-color: {{$category->color}};">
-                        <img src="{{ asset('image/category' .$category->icon) }}" alt="" class="card-img-top" style="height: 80px; width:80px;"></a>
+                        <img src="{{ asset('image/category/' .$category->icon) }}" alt="" class="card-img-top" style="height: 80px; width:80px;"></a>
```
Comment (krs91tk) [inline resources/views/users/top.blade.php:19]: pls move to this to CSS file if you can
`style="height: 80px; width:80px;"`

File: resources/views/users/reserved/join_users.blade.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                <div class="modal fade" id="other-member{{ $user->id }}">
+                    <div class="modal-dialog">
+                        <div class="modal-content border-warning">
+                            <div class="modal-header">
+                                <div class="modal-title mx-auto">OTHER MEMBER</div>
+                            </div>
+                            <div class="modal-body">
+                                <div class="avatar mx-auto">
+                                    @if($user->avatar)
+                                    <img src="{{ asset('storage/avatars/'.$user->avatar) }}" alt="" class="avatar-md">
```
Comment (krs91tk) [inline resources/views/users/reserved/join_users.blade.php:43]: If you dont need this pls delete
`alt=""`

File: resources/views/users/reserved/join_users.blade.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                                    @else
+                                    <i class="fa-solid fa-circle-user fa-2xl text-secondary text-center "></i>
```
Comment (krs91tk) [inline resources/views/users/reserved/join_users.blade.php:45]: delete space after `text-center`

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (yumeR07) [reply app/Http/Controllers/HomeController.php:22]: No need. So I delete it.

File: resources/views/users/top.blade.php (hunk 2 above)
Comment (yumeR07) [reply resources/views/users/top.blade.php:19]: I added this style in users-style.scss.

File: resources/views/users/reserved/join_users.blade.php (hunk 3 above)
Comment (yumeR07) [reply resources/views/users/reserved/join_users.blade.php:43]: I delete it.

File: resources/views/users/reserved/join_users.blade.php (hunk 4 above)
Comment (yumeR07) [reply resources/views/users/reserved/join_users.blade.php:45]: I delete this space.

**Pull request merged**